src/components/peerstatsview/QualityIndicator.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import { useContext } from 'react';
+import { ServiceContext } from '../../store/store';
+import { AnomalyDetector } from '../../utils/anomalyDetector';
+
+interface QualityIndicatorProps {
+	peerConnectionLabel?: string;
+}
+
+const QualityIndicator = ({
+	peerConnectionLabel,
+}: QualityIndicatorProps): JSX.Element => {
+	const { mediaService } = useContext(ServiceContext);
+	const [ isDistorted, setIsDistorted ] = useState<boolean>(false);
+
+	useEffect(() => {
+		// this runs on mount
+		const monitor = mediaService.getMonitor();
+		
+		if (!monitor) {
+			return;
+		}
+
+		const storage = monitor.storage;
+		const windowSize = 30;
+		const stdMultiplierThreshold = 3;
+		const minDeviationThreshold = 0.05;
+		const detector = new AnomalyDetector(windowSize, stdMultiplierThreshold, minDeviationThreshold);
+
+		const listener: () => void = () => {
```

Comment:
This is never used anywhere, or am I missing something?

Reply:
yup, you are right